client/users-form.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import Paper from 'material-ui/Paper'
+import TextField from 'material-ui/TextField'
+import DatePicker from 'material-ui/DatePicker'
+import RaisedButton from 'material-ui/RaisedButton'
+
+export default function UsersForm({ style, handleSubmit }) {
+  return (
+    <Paper style={style} zDepth={1}>
+      <h1>Create an Account!</h1>
+      <form onSubmit={handleSubmit}>
+        <TextField
+          hintText="Please type a username."
+          floatingLabelText="Username"
+          type="text"
+          name="username"
+          fullWidth = {true}
+        /><br />
```

Comment:
If you want elements to stack, it's best to wrap them in a block-level element instead of throwing in `<br/>` elements.